src/index.js

```diff
@@ -1,7 +1,33 @@
 import React from 'react';
 import { render } from 'react-dom';
+import {
+  Route,
+  NavLink,
+  BrowserRouter as Router,
+  Switch,
+} from 'react-router-dom';
 import App from './App';
+import Notfound from './NotFound';
+
+const routing = (
+  <Router>
+    <div>
+      <ul>
+        <li>
+          <NavLink exact activeClassName="active" to="/">
+            Home
+          </NavLink>
+        </li>
+      </ul>
+
+      <Switch>
+        <Route exact path="/" component={App} />
+        <Route component={Notfound} />
+      </Switch>
+    </div>
```

Comment:
I recommend you to use `index.js` file to render our application and `App.js` to import all components and configure routes.

``` js
// index.js
ReactDOM.render(<App />, document.getElementById('root'));

//App.js
const App = () => {
  return (
      <Router>
        <div className="container">
          <Switch>
            <Route path="/" exact component={componentA} />
            <Route path="/signin" exact component={componentB} />
            <Route path="/signup" exact component={componentC} />
          </Switch>
        </div>
      </Router>
  );
}

```